### ISSUE TITLE
Variant inventory table header should stay fixed as I scroll

### ISSUE THREAD

**Issue by morebrownies** [OpenBazaar/openbazaar-desktop#417]
I find myself frequently scrolling back up to view the column titles in the variant inventory section. It gets especially complicated when there are a lot of variants and choices.

Currently, when I scroll, I see: (table header scrolls off screen)
![screenshot 2017-03-24 10 35 32](https://cloud.githubusercontent.com/assets/13066886/24306542/3091afc6-107e-11e7-9eda-3260c89a046e.png)

Ideally, the table header would stay fixed as I scroll: 
![screenshot 2017-03-24 10 47 14](https://cloud.githubusercontent.com/assets/13066886/24306848/454beed0-107f-11e7-8e43-07cc220932ee.png)

**Comment by BillStrait** [comment]
I have this done. I did a bad job with my change management, so it's waiting on another PR before I submit it.

**Comment by morebrownies** [comment]
great! thank you @BillStrait!
